ckanext/protected_resources/plugin.py

```diff
@@ -156,20 +136,37 @@ def auth_protected_resource_lock(context, data_dict=None):
     else:
```

Comment:
2/10
This `else` is not necessary

Reply:
I don't think it hurts clarity though, so I probably won't fix this